docker/dispatcher-image/Dockerfile

```diff
@@ -41,6 +39,9 @@ RUN curl https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 > \
     /usr/local/bin/cloud_sql_proxy
 RUN chmod +x /usr/local/bin/cloud_sql_proxy
 
+# Add libClusterFuzz for stacktrace parsing.
+RUN pip3 install clusterfuzz
```

Comment:
Why can't we put this in requirements.txt?
Linting will explode if we start using this package without doing that.

Reply:
Sure, will do.